### DISCUSSION TITLE
Sh1mmer help

### DISCUSSION THREAD

**Discussion by doofenator** [3kh0/ext-remover#803] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm trying to sh1mmer my chromebook but i'm not sure where to start and how to acquire a shim, how to modify it, etc. My board's nissa, could somebody please help me out

**Comment by Yeetusmans2048** [comment, marked as answer]
I know this question is old, but here
Modern: https://dl.darkn.bio/SH1mmer/Prebuilt/Modern/modern_nissa.zip
Legacy: https://dl.darkn.bio/SH1mmer/Prebuilt/Legacy/legacy_nissa.zip

**Reply by doofenator** [reply]
Thank you, unfortunately my kernel is already version 3 :/
Really useful tho, I'll keep this around for 100% legal reasons, such as not spreading this around my school

**Reply by 3kh0** [reply]
![xdd](https://github.com/3kh0/ext-remover/assets/58097612/1c85f2e7-d316-4b43-9f0b-b0161b3c559b)

**Reply by Yeetusmans2048** [reply]
> Thank you, unfortunately my kernel is already version 3 :/ Really useful tho, I'll keep this around for 100% legal reasons, such as not spreading this around my school

If your kernel version is 3 use [THIS GUIDE](https://blog.darkn.bio/blog/3-the-tsunami)

**Comment by D3vToo1s** [comment]
Get free nissa sh1mmer sh1ms here https://www.dropbox.com/t/nGMnN2Qa6D0Hq2jk
